clifford/dpga.py

```diff
@@ -0,0 +1,64 @@
+
+""" 
+The Cl(4,4) DPGA also known as `Mother Algebra' of Goldman and Mann. see:
+
+R(4, 4) As a Computational Framework for 3-Dimensional Computer Graphics
+Ron Goldman and Stephen Mann
+
+and
+
+Modeling 3D Geometry in the Clifford Algebra R(4, 4)
+Juan Du, Ron Goldman & Stephen Mann
+
+and
+
+Transverse approach to geometric algebra models for manipulating quadratic surfaces
+Stéphane Breuils, Vincent Nozick, Laurent Fuchs, Akihiro Sugimoto
+June 2019 Calgary Canada
+"""
+
+from . import Cl
+import numpy as np
+
+layout, blades = Cl(4, 4)
```

Comment:
I'd recommend
```suggestion
layout = Layout([1]*4 + [-1]*4, ids=BasisVectorIds(['{}'.format(i) for i in range(4)] + ['{}b'.format(i) for i in range(4)])
blades = layout.bases()
```
Then you never see `e5`, `e6` etc at all.